index.html

```diff
@@ -24,9 +24,10 @@
         {url: "src/main/resources/openapi/cases.yaml", name: "Cases"},
         {url: "src/main/resources/openapi/courthouse.yaml", name: "Courthouses"},
         {url: "src/main/resources/openapi/dailylist.yaml", name: "Daily Lists"},
-        {url: "src/main/resources/openapi/noderegistration.yaml", name: "Devices"},
         {url: "src/main/resources/openapi/event.yaml", name: "Events"},
         {url: "src/main/resources/openapi/hearings.yaml", name: "Hearings"},
+        {url: "src/main/resources/openapi/noderegistration.yaml", name: "Devices"},
+        {url: "src/main/resources/openapi/retentions.yaml", name: "Retentions"},
```

Comment:
We probably need to remember to add the `SecurityRequirement` annotation to the Controller endpoint, so that we can use the application swagger-ui Try it out functionality